### ISSUE TITLE
FIMSFrameAGE is being deprecated in m2-r-cleanup

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [NOAA-FIMS/case-studies#25]
FIMSFrameAge will have the same functionality as FIMSFrame and can be renamed as such. This affects 
* [content/NWFSC-petrale.qmd](https://github.com/NOAA-FIMS/case-studies/blob/6f938ad21c04aeddfe1aa12f644ded8a9816c215/content/NWFSC-petrale.qmd#L93-L94)
* [content/R/get_ss3_data.R](https://github.com/NOAA-FIMS/case-studies/blob/6f938ad21c04aeddfe1aa12f644ded8a9816c215/content/R/get_ss3_data.R#L9)
* [content/AFSC-BSAI-AtkaMackerel.qmd](https://github.com/NOAA-FIMS/case-studies/blob/6f938ad21c04aeddfe1aa12f644ded8a9816c215/content/AFSC-BSAI-AtkaMackerel.qmd#L170)
* [content/NEFSC-yellowtail.qmd](https://github.com/NOAA-FIMS/case-studies/blob/6f938ad21c04aeddfe1aa12f644ded8a9816c215/content/NEFSC-yellowtail.qmd#L147)
* [content/R/pk_prepare_FIMS_inputs.R](https://github.com/NOAA-FIMS/case-studies/blob/6f938ad21c04aeddfe1aa12f644ded8a9816c215/content/R/pk_prepare_FIMS_inputs.R#L156)
* [content/NWFSC-petrale.R](https://github.com/NOAA-FIMS/case-studies/blob/6f938ad21c04aeddfe1aa12f644ded8a9816c215/content/NWFSC-petrale.R#L53-L54)
but the timing of the change will have to coincide with the merge of m2-r-cleanup into main, where first m2-r-cleanup-remove will be merged into m2-r-cleanup and then by the end of next week m2-r-cleanup should be merged into main.
@iantaylor-NOAA, @MOshima-PIFSC, or @Bai-Li-NOAA are you willing to take this on?

See https://github.com/NOAA-FIMS/FIMS/issues/590 for the main issue.

**Comment by iantaylor-NOAA** [comment]
I can do the global find and replace. It will be a pleasure to have only one function.

If it's OK with the authors, I will modify any comments as well, such as replacing (for petrale)
```
age_frame <- FIMS::FIMSFrameAge(mydat) # similar to FIMSFrame() but includes ages
# in the future FIMSFrame() and FIMSFrameAge() will likely be merged
```
with
```
age_frame <- FIMS::FIMSFrame(mydat) 
```

We should be able to make the timing work well but it's also not a big deal if case-studies breaks for a bit.

**Comment by kellijohnson-NOAA** [comment]
Thanks @iantaylor-NOAA!

**Comment by kellijohnson-NOAA** [comment]
@iantaylor-NOAA this change is now in main with [2431f59](https://github.com/NOAA-FIMS/FIMS/commit/2431f59af3f0895e8b305ce30006d50c8d799355) so the case studies can be modified.

**Comment by iantaylor-NOAA** [comment]
Resolved by #26.
